src/test/test_utils.py

```diff
@@ -38,11 +39,17 @@
                           ("wav", ["audio/x-wav"]),
                           ("wma", ["audio/x-ms-wma", "video/x-ms-wma",
                                    "video/x-ms-asf"])])
+
 def testSampleMimeTypes(ext, valid_types):
```

Comment:
This new line should be removed, the fixture decorator above the function is applied to this function.